drivers/soundwire/bus.c

```diff
@@ -1239,20 +1239,27 @@ static int sdw_handle_slave_alerts(struct sdw_slave *slave)
 
 	sdw_modify_slave_status(slave, SDW_SLAVE_ALERT);
 
+	ret = pm_runtime_get_sync(&slave->dev);
```

Comment:
Also typos and grammar mistakes in the commit message. 
"There are two types of io error*s* when processing alert event.
The first one is: master receives a*n* alert event for jack event
and invokes implement_def function in slave to check jack status.
At this time codec is just suspended, so io registers can't be
accessed. Another one is: when w*o*ken *or waking* up from clock stop state
and bus needs a complete re-enumeration for synchronization issue
, slave register can't be accessed."

Reply:
I did not modify this commit message since it's a controversial patch, with errors reported in https://github.com/thesofproject/linux/issues/1597

@RanderWang we really need to figure out if this is needed or not.

Reply:
@plbossart   I showed the details in https://github.com/thesofproject/linux/pull/1589. Thanks!

Reply:
> implement_def

Thanks! @ranj063  I fixed in https://github.com/thesofproject/linux/pull/1589